compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -802,6 +811,12 @@ convertExpr env0 e = do
         t2' <- convertType env t2
         pure (x' `ETyApp` t1' `ETyApp` t2' `ETmApp` EBuiltin (BEText (unpackCStringUtf8 s)))
 
+    go env (ConstraintTupleProjection i j) args | (LExpr x : args') <- drop j args =
```

Comment:
Are the `arity` arguments that we drop the types? If so, it would be good to add a comment since I had to spend 5 minutes staring at this to figure it out :slightly_smiling_face:

Reply:
Yes, will do!